packages/template-retail-react-app/app/hooks/use-datacloud.js

```diff
@@ -252,6 +287,14 @@ export class DataCloudApi {
                   userDetails
               )
 
+        const partyIdentification = this.dnt
+            ? {}
+            : this._concatenateEvents(
+                  baseEvent,
+                  this._generateEventDetails('partyIdentification', 'Profile'),
+                  this._constructPartyIdentification(args)
+              )
```

Comment:
This block of code is repeated for every send-related methods. 

I see how each of those methods already contain repeated code. So without a refactor, I guess this duplicate code is fine for now.